nearlib/signing/browser_local_storage_key_store.js

```diff
@@ -27,27 +31,37 @@ class BrowserLocalStorageKeystore {
      * @param {KeyPair} key 
      */
     async setKey(accountId, key) {
-        window.localStorage.setItem(
+        this.localStorage.setItem(
             BrowserLocalStorageKeystore.storageKeyForPublicKey(accountId), key.getPublicKey());
-        window.localStorage.setItem(
+        this.localStorage.setItem(
             BrowserLocalStorageKeystore.storageKeyForSecretKey(accountId), key.getSecretKey());
     }
 
+    async setKeyFromJson(json) {
+        const accountInfo = new JsonAccountInfo(json);
+        if (this.networkId != accountInfo.getNetworkId()) {
+            throw 'Setting key for a wrong network';
```

Comment:
```js
throw new Error('Setting key for a wrong network')
```

Reply:
(same applies to other cases of throwing errors)

Reply:
done